ocsmesh/driver.py

```diff
@@ -71,14 +72,24 @@ def __init__(
         """
         geom can be SizeFunction or PlanarStraightLineGraph instance.
         """
+        warnings.warn(
+            "JigsawDriver is deprecated and will be removed in a future release. "
+            "Please use 'ocsmesh.driver.MeshDriver' with engine_name='jigsaw'." 
+            "We strongly suggest the adoption of gmsh and the driver engine",
+            FutureWarning, stacklevel=2
+        )
         self._geom = geom
         self._hfun = hfun
         self._init = initial_mesh
         self._crs = CRS.from_user_input(crs) if crs is not None else crs
         self._opts.verbosity = verbosity
 
     def run(self, sieve=None, quality_metric=1.05, remesh_tiny_elements=False):
-
+        warnings.warn(
+            "JigsawDriver.run() is deprecated. "
+            "Use MeshDriver.run() instead.",
```

Comment:
Again I'm not sure if having a reference to something that will be added later is going to be helpful or not